cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -60,7 +63,12 @@ describe('Creator Settings Page', () => {
       'have.attr',
       'required',
     );
-    cy.findByRole('button', { name: /logo/i }).should('have.attr', 'required');
+
+    cy.findByLabelText(/upload logo/i, { selector: 'input' }).should(
+      'have.attr',
+      'required',
+    );
```

Comment:
One thing I noticed is that although this field is required, there's no visual indication of that other than the button focusing after submit click:

https://user-images.githubusercontent.com/20773163/141985225-128ac1eb-ba80-406f-9902-950dc4c63c6e.mp4

If every field is required I wonder if we should consider adding a note on the page that says that (it's just a thought 🤷‍♀️ )

Reply:
Ahh, another good catch. This is because we're using default HTML validation and since the actual input is not visible, its validation message never appears. Here it is when it's visible. 

![image](https://user-images.githubusercontent.com/833231/142022290-237f7aca-877f-489d-8a43-87d16a26b73e.png)

I think the note makes sense, or I can see how I can make it appear. 🤔